task_5/1.js

```diff
@@ -0,0 +1,28 @@
+// Complete the solution so that the function will break up camel casing, using a space between words.
+
+// fn("camelCasing")  //  "camel Casing"
+// fn("identifier")  //  "identifier"
+// fn(‘helloMyDearFriend’) // ’hello My Dear Friend’
+
+const decodeCamelCaseString = (anyString) => {
+    if(typeof anyString !== 'string') return `It's not a string.`;
+    if(anyString.length < 2) return anyString;
+
+    const lettersArray = anyString.split('');
+    let resultString = ``;
+    let i = 0;
+
+    while(i < lettersArray.length) {
+        const letter = lettersArray[i];
+        const capitalLetter = letter.toUpperCase();
+        ((letter === capitalLetter) && (i !== 0)) ? resultString += ` ${letter}` : resultString += `${letter}`;
+        i++;
+    }
```

Comment:
1. you don't need to check first letter because camelCase always starts with the lower one
2. Why you use while loop? it's outdated stuff. Use modern methods